packages/wonder-blocks-tokens/src/theme/semantic/semantic-color.ts

```diff
@@ -725,7 +721,7 @@ export const semanticColor = {
         rest: core.foreground.instructive.subtle,
         hover: core.foreground.instructive.subtle,
         press: core.foreground.instructive.default,
-        disabled: core.foreground.inverse.subtle,
+        disabled: core.foreground.disabled.default,
```

Comment:
`foreground.inverse.subtle` was `fadedOffBlack32`, `foreground.disabled.default` is also `fadedOffBlack32`!